src/aegis_ai/__init__.py

```diff
@@ -70,7 +65,25 @@ class AppSettings(BaseSettings):
     default_llm_host: str = llm_host
     default_llm_model: str = llm_model
     default_llm_settings: Dict = default_llm_settings
-    otel_enabled: bool = otel_enable
+
+    otel_enabled: bool = os.getenv("AEGIS_OTEL_ENABLED", "false").lower() in (
+        "true",
+        "1",
+        "t",
+        "y",
+        "yes",
+    )
+
+    safety_enabled: bool = os.getenv("AEGIS_SAFETY_ENABLED", "false").lower() in (
```

Comment:
I would insert a comment to explain that this only takes an effect if `prompt.is_safe()` is explicitly called.